database/gormdb_assignment_test.go

```diff
@@ -48,26 +47,27 @@ func TestGormDBCreateAssignment(t *testing.T) {
 	admin := qtest.CreateFakeUser(t, db, 10)
 	qtest.CreateCourse(t, db, admin, &pb.Course{})
 
-	assignment := pb.Assignment{
+	gotAssignment := pb.Assignment{
```

Comment:
Here you should initialize `gotAssignment` using `&pb.Assignment{` and remove the `&` on line 55. This is because otherwise we will get complaints from the linter that we are copying the mutex lock (see the warning from the linter on line 69). All the `pb.*` types are protobuf types and these contain a mutex lock... But it is easy to avoid the problem by always using a pointer.

Reply:
Another option is to pass the argument as a pointer to `cmp.Diff`. However, I would prefer to use the style: `&pb.Assignment{`...